src/Makefile.am

```diff
@@ -64,10 +65,15 @@ DISPATCH_CFLAGS=-Wall $(VISIBILITY_FLAGS) $(OMIT_LEAF_FP_FLAGS) \
 if DISPATCH_ENABLE_ASSERTS
 DISPATCH_CFLAGS+=-DDISPATCH_DEBUG=1
 endif
+
+if ANDROID
+ANDROID_LIBS= -L$(top_srcdir)/src/shims/android
```

Comment:
Trust me, I'm not proud about them. The submodules are not in the `apple` organization, so I wasn't sure how that may affect the chances + delay to include this. I'll start working on that tho.